File: content/code-examples/tested/javascript/driver/time-series/auto-removal.snippet.create-timeseries-collection-for-removal.js
const createCommand = {
  create: 'weather24h',
  timeseries: {
    timeField: 'timestamp',
    metaField: 'sensorId',
    granularity: 'seconds',
  },
  expireAfterSeconds: 86400,
};

// Execute the command to create the collection
await database.command(createCommand);

File: content/code-examples/tested/csharp/driver/TimeSeries/AutoRemoval.snippet.create-timeseries-collection-for-removal.cs
var createCommand = new BsonDocument
{
    { "create", "weather24h" },
    { "timeseries", new BsonDocument
        {
            { "timeField", "timestamp" },
            { "metaField", "sensorId" },
            { "granularity", "seconds" }
        }
    },
    { "expireAfterSeconds", 86400 }
};

// Execute the command to create the collection  
await database.RunCommandAsync<BsonDocument>(createCommand);
